cmd/enumerate_github/README.md

```diff
@@ -0,0 +1,130 @@
+# GitHub Enumeration Tool
+
+This tool is used to reliably enumerate projects on GitHub.
+
+The output of this tool is used as an input for the `collect_signals` tool. 
+
+## Example
+
+```shell
+$ export GITHUB_TOKEN=ghp_x  # Personal Access Token Goes Here 
+$ enumerate_github \
+    -start 2008-01-01 \
+    -min-stars=10 \
+    -workers=1 \
+    github_projects.txt
+```
+
+## Install
+
+```shell
+$ go install github.com/ossf/criticality_score/cmd/enumerate_github
+```
+
+## Usage
+
+```shell
+$ enumerate_github [FLAGS]... FILE
+```
+
+The URL for each repository is written to `FILE`. If `FILE` is `-` the results will be written to STDOUT.
+
+### Flags
```

Comment:
Are any flags required vs optional? Might be good to add a line on that.

Reply:
Added.